test/hotspot/jtreg/containers/docker/TestMemoryAwareness.java

```diff
@@ -153,18 +153,24 @@ private static void testOperatingSystemMXBeanAwareness(String memoryAllocation,
         out.shouldHaveExitValue(0)
            .shouldContain("Checking OperatingSystemMXBean")
            .shouldContain("OperatingSystemMXBean.getTotalPhysicalMemorySize: " + expectedMemory)
-           .shouldMatch("OperatingSystemMXBean\\.getFreePhysicalMemorySize: [1-9][0-9]+")
            .shouldContain("OperatingSystemMXBean.getTotalMemorySize: " + expectedMemory)
            .shouldMatch("OperatingSystemMXBean\\.getFreeMemorySize: [1-9][0-9]+")
-           .shouldMatch("OperatingSystemMXBean\\.getFreeSwapSpaceSize: [1-9][0-9]+");
-        // in case of warnings like : "Your kernel does not support swap limit capabilities or the cgroup is not mounted. Memory limited without swap."
-        // the getTotalSwapSpaceSize returns the system values as the container setup isn't supported in that case.
+           .shouldMatch("OperatingSystemMXBean\\.getFreePhysicalMemorySize: [1-9][0-9]+");
+
+        // in case of warnings like : "Your kernel does not support swap limit capabilities
+        // or the cgroup is not mounted. Memory limited without swap."
+        // the getTotalSwapSpaceSize does not return the expected result and
+        // getFreeSwapSpaceSize returns 0
```

Comment:
https://bugs.openjdk.java.net/browse/JDK-8244500 indicates a system where we have the kernel warning, but getFreeSwapSpaceSize() returned the system values. *Not* 0 as indicated in this comment. This yields me to believe there are inconsistently behaving systems out there. Could we rephrase this comment?

Reply:
Thanks for reviewing this!
Does this change to the comment look better?

         // in case of warnings like : "Your kernel does not support swap limit capabilities
         // or the cgroup is not mounted. Memory limited without swap."
-        // the getTotalSwapSpaceSize does not return the expected result and
-        // getFreeSwapSpaceSize returns 0
+        // the getTotalSwapSpaceSize and getFreeSwapSpaceSize return the system
+        // values as the container setup isn't supported in that case.

Reply:
This looks fine. Thanks!